### ISSUE TITLE
Maintenance testing for fractions sims

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/QA#448]
I'd like spot-checks for the following fractions sims, to make sure that the recent maintenance patches work nicely (so they can be production deployed sometime):

- [ ] https://phet-dev.colorado.edu/html/fraction-matcher/1.2.0-rc.4/phet/fraction-matcher_en_phet.html
- [ ] https://phet-dev.colorado.edu/html/fractions-equality/1.1.0-rc.4/phet/fractions-equality_en_phet.html

This should include iOS-related testing (iPadOS glitches, full-screen, game audio tests)

The last RC testing was done in:

- https://github.com/phetsims/QA/issues/413
- https://github.com/phetsims/QA/issues/414

And the sim-specific patch issues (auto-generated by the maintenance release process):

- https://github.com/phetsims/fraction-matcher/issues/109
- https://github.com/phetsims/fractions-equality/issues/7

Additionally @ariel-phet, should I proceed with production deploys once these spot checks are finished? I don't see any non-deferred issues from the earlier RCs or in any relevant repos.

**Comment by KatieWoe** [comment]
- [ ] Win 10 chrome
- [ ] Win 10 firefox
- [ ] Win 10 edge
---
- [ ] Mac 10.15 safari
---
- [ ] Win 7 IE
---
- [ ] ChromeOS
- [ ] iPadOS 13

**Comment by ariel-phet** [comment]
high priority since should be quick but after #445 (or concurrently with #445 if on the same platform)

**Comment by KatieWoe** [comment]
QA is done

**Comment by jonathanolson** [comment]
Thanks, moving forward with production deployments!
